release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2775,6 +2775,16 @@ After you run the command, the CNO collects `must-gather` logs that you can insp
 +
 (link:https://issues.redhat.com/browse/OCPBUGS-52367[OCPBUGS-52367])
 
+* There is a known issue with Gateway API that prevents the provisioning of internal load balancers on {aws-first}, {gcp-first}, and {azure-first} private clusters. Load balancers configured for internal use are incorrectly determined to be external:
```

Comment:
It is confusing to say, "Load balancers configured for internal use are incorrectly determined to be external"; the issue is that we don't provide an option to configure the load balancer to be internal.  
```suggestion
* There is a known issue with Gateway API and {aws-first}, {gcp-first}, and {azure-first} private clusters.  The load balancer that is provisioned for a gateway is always configured to be external, which can cause errors or unexpected behavior:
```

Reply:
thanks Miciah, i've updated the text